css/styles.css

```diff
@@ -0,0 +1,839 @@
+  padding-left: 25px;
+  padding-right: 25px;
+  padding-top: 5px;
+  text-align: center;
+  font-weight: 400;
+}
+
+.social {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  list-style: none;
+  padding: 35px 25px 20px 25px;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.social li {
+  margin-left: 10px;
+  margin-right: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.arrow-wrapper img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  transition: all 0.5s;
+}
+
+.social li img {
+  display: block;
+}
+
+.arrow-wrapper img:hover {
+  transform: translateY(8px);
+  transition: all 0.5s;
+}
+
+.banner-section {
+  text-align: center;
+  max-width: 710px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.banner-section p {
+  padding-left: 24px;
+  padding-right: 24px;
+  font-size: 20px;
+  line-height: 28px;
+  letter-spacing: -0.37px;
+}
+
+@media (min-width: 768px) {
+  .banner-section p {
+    font-size: 18px;
+  }
+
+  h1 {
+    padding-top: 12%;
+  }
+}
+
+.arrow-wrapper {
+  padding-top: 30px;
+  display: inline-block;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.headlined {
+  display: flex;
+  flex-direction: column;
+
+  /* border: 2px solid red; */
+  width: 920px;
+  height: 756px;
+  top: 72px;
+  margin: auto;
+}
+
+.headlinecontents {
+  height: 114px;
+  width: 920px;
+  text-align: center;
+}
+
+.headlinecontents h4 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 48px;
+  line-height: 60px;
+  letter-spacing: 0.4px;
+  color: #172b4d;
+}
+
+.green {
+  color: green;
+}
+
+.text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #172b4d;
+}
+
+.ul-socialmedia {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: auto;
+  width: 200px;
+  height: 40px;
+  margin-top: 28px;
+  padding-inline-start: 0;
+}
+
+.socialists {
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.sec {
+  width: 40px;
+  height: 40px;
+  margin-top: 30px;
+  border-radius: 32px;
+  border: 1px solid #dfe1e6;
+}
+
+.down {
+  width: 14px;
+  height: 8px;
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  padding: 80px, 0, 80px, 0;
+  gap: 48px;
+  margin: auto;
+}
+
+.work-text {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  text-align: center;
+  margin: auto;
+  max-width: 447px;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+#work-grid {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: (minmax(280px, 370px)) [auto-fit];
+  grid-template-columns: repeat(auto-fit, minmax(280px, 370px));
+  grid-gap: 25px;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
```

Comment:
- [ ] The projects grid should be set to show only 3 projects per row, as per the design of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468), kindly fix that to match the design. 😁

![image](https://user-images.githubusercontent.com/20567503/178080547-c1cfc42d-a4a1-4b9e-ba19-0ddcaed7b8db.png)

Note: please also be mindful of the horizontal paddings and margins of this section. 😉